src/asset/img/textured-bg-mobile.svg

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I don't quite understand why there's a `.js` and ` .svg` file for every background..

Reply:
Because we might need the source .svg in the future and someone might not have access to sketch e.g. not on macOS.